### PR TITLE
docs landing page urgent tweaks

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Adjusts card links and coloring -- open to suggestions, but this is an urgent patch fix.

## When should this change go live?

- [x] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [ ] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by akeller** [comment]
I have added the new `deploy` tag Infra has been working on so you can see it on a preview site to solicit feedback faster/easier.

File: src/css/custom.css
Comment (conceptualshark) [inline src/css/custom.css:1]: I understand you've probably received more direct feedback which might influence this (and want to merge fast), so feel free to disregard! 

Some quick options for cleaning up the feature cards:
```.component-block {
  transition: all 0.3s ease;
  border-radius: 25px;
  box-shadow: 4px 4px 4px 1px lightgray;
  margin: 2px;
}

.component-block:hover {
  transform: translateY(-8px);
  transition: all 0.3s ease;  
}

.component-block a {
  color: var(--ifm-color-black);
}

.component-block a:hover {
  color: var(--ifm-color-primary);
  text-decoration: none;
}``` 
<img width="1335" alt="Screenshot 2024-07-16 at 12 19 11 PM" src="https://github.com/user-attachments/assets/5e65fea8-4415-46a7-b89f-e2f2576345c7">

Or, if we're hoping to add the underline for clear link accessibility reasons:

**Comment by christinaausley** [comment]
> I have added the new deploy tag Infra has been working on so you can see it on a preview site to solicit feedback faster/easier.

Is this something we are trying to urgently merge in? I was planning to solicit feedback this week as @PhilippaC22 and I get started on a formal second iteration.

**Comment by akeller** [comment]
> Is this something we are trying to urgently merge in? I was planning to solicit feedback this week as @PhilippaC22 and I get started on a formal second iteration, but thought these changes were rushed specifically for today.

You should still get quick feedback on urgent changes, but at a certain point you can stop accepting feedback and document that you will move forward and iterate in the future. Particularly in this case, since you are making UX/UI changes and are open to suggestions, we should make it easy to see the changes.

**Comment by akeller** [comment]
Well... I thought the preview site would be great, but it looks like it's on an old commit. When running locally, these are the changes I see (and the UX problem where the button/box isn't entirely clickable is fixed!): 


https://github.com/user-attachments/assets/ff339084-a56a-46f5-9b74-5e6ab131898b

**Review by akeller — APPROVED** [review]
The UX issue appears to be addressed, the whole box is clickable!

**Pull request merged**